gradle/versions.gradle

```diff
@@ -13,17 +13,20 @@ ext {
     android_x_constraint_layout = '1.1.3'
     android_x_multidex = '2.0.1'
 
-    vision_native = '0.3.0'
-    vision_sdk = '0.3.0'
-    vision_ar = '0.3.0'
-    vision_safety = '0.3.0'
+    vision = '0.4.0'
+    spek_version = '1.0.0'
```

Comment:
spek version is 2.0.4 and please move it to the bottom of file

Reply:
Riiight, again - debug, wasn't really intended to push this yet.